source/chapters/library.rst

```diff
@@ -691,25 +719,22 @@ If however you want to plan out a specific set and practice the transitions,
 you will want an ordered list of tracks. Playlists serve this purpose.
 
 
-.. _library-browse:
+.. _library-computer:
 
-Browse - Loading remote tracks
-==============================
+Computer - Loading remote tracks
```

Comment:
"remote" is an odd word here. What about "Loading tracks from anywhere on your computer" ?